frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -239,13 +200,13 @@ const QueueItem = ({
   };
 
   const [mutationError, updateMutationError] = useState(null);
-  const [removePatientFromQueue] = useMutation(REMOVE_PATIENT_FROM_QUEUE);
-  const [submitTestResult, { loading }] = useMutation(SUBMIT_TEST_RESULT);
+  const [removePatientFromQueue] = useRemovePatientFromQueueMutation();
```

Comment:
Thank you for editing these other mutations to be in the new style 🙏

Reply:
Oh thanks for doing this, Johanna! and also pointing this out, Emma! will be sure to do this in my PRs too 😅